tests/typ/meta/query.typ

```diff
@@ -70,3 +70,84 @@
   image("/tiger.jpg"),
   caption: [Tiger world],
 )
+
+---
```

Comment:
the reference image for the test is too big (query already rendered quite slowly before). could you split this up into smaller tests, ideally with less pixels (by having e.g. smaller page size, etc.)